python/paddle/fluid/layers/tensor.py

```diff
@@ -448,13 +448,13 @@ def assign(input, output=None):
     The OP copies the :attr:`input` to the :attr:`output`.
 
     Parameters:
-        input (Variable|numpy.ndarray): A tensor or numpy ndarray, its data type supports
+        input (Variable|numpy.ndarray): A tensor, numpy ndarray or LoDTensorArray, its data type supports
```

Comment:
A better order will be "A tensor, or LoDTensorArray, or numpy ndarray"

Reply:
Done.

Reply:
- Tensor and LoDTensorArray are belong to Variable. So, "A Tensor, or LoDTensorArray, or numpy ndarray" is better
- tensor -> Tensor